deployer/src/persistence/mod.rs

```diff
@@ -392,35 +338,22 @@ impl Persistence {
         .map_err(Error::from)
     }
 
-    /// Get a broadcast channel for listening to logs that are being stored into persistence
-    pub fn get_log_subscriber(&self) -> Receiver<deploy_layer::Log> {
-        self.stream_log_send.subscribe()
-    }
-
-    /// Returns a sender for sending logs to persistence storage
-    pub fn get_log_sender(&self) -> crossbeam_channel::Sender<deploy_layer::Log> {
-        self.log_send.clone()
-    }
-
     pub async fn stop_running_deployment(&self, deployable: DeploymentRunnable) -> Result<()> {
         update_deployment(
             &self.pool,
             DeploymentState {
                 id: deployable.id,
-                last_update: Utc::now(),
                 state: State::Stopped,
             },
         )
         .await
     }
 }
 
-async fn update_deployment(pool: &SqlitePool, state: impl Into<DeploymentState>) -> Result<()> {
-    let state = state.into();
-
+async fn update_deployment(pool: &SqlitePool, state: DeploymentState) -> Result<()> {
     sqlx::query("UPDATE deployments SET state = ?, last_update = ? WHERE id = ?")
         .bind(state.state)
-        .bind(state.last_update)
+        .bind(Utc::now())
```

Comment:
Oh, okay. :thinking: I'm not sure if this might be delayed though

Reply:
Not in the two places it is currently called from at least.